README.md

```diff
@@ -31,7 +31,7 @@ make ui
 
 ## Building Backend
 ```
-make backend
+export GO111MODULE=on && make backend
```

Comment:
https://maelvls.dev/go111module-everywhere/#go111module-with-go-116
`As of Go 1.16, the default behavior is GO111MODULE=on` - not sure that we need to add that